features/claim/wallet.tsx

```diff
@@ -69,10 +73,13 @@ export const Wallet: FC = () => {
               <>
                 <div>
                   <TokensAmount>
-                    <FormatToken amount={unclaimed} symbol={token?.symbol} />
+                    <FormatToken
+                      amount={unclaimed}
+                      symbol={token?.symbol || ''}
+                    />
                   </TokensAmount>
                   &nbsp;
-                  <TokenToWallet address={token?.address} />
+                  <TokenToWallet address={token?.address || ''} />
```

Comment:
Why can't it be undefined?

Reply:
needs patch to TokenToWallet (warehouse)

Reply:
done